libs/novui/src/json-schema-components/widgets/InputEditorWidget.tsx

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useMemo, useRef } from 'react';
+
+import { WidgetProps } from '@rjsf/utils';
+
+import { Input } from '@mantine/core';
+import { RichTextEditor } from '@mantine/tiptap';
+
+import { type Extensions, useEditor } from '@tiptap/react';
+import Text from '@tiptap/extension-text';
+import Paragraph from '@tiptap/extension-paragraph';
+import { ReactRenderer } from '@tiptap/react';
+import Document from '@tiptap/extension-document';
+import Mention from '@tiptap/extension-mention';
+
+import { css, cx } from '../../../styled-system/css';
+import { input } from '../../../styled-system/recipes';
+import { splitCssProps } from '../../../styled-system/jsx';
+import classes from './InputEditorWidget.styles';
+
+import { VariableSuggestionList, SuggestionListRef, VariableItem } from './VariableSuggestionList';
+
+import '@mantine/tiptap/styles.css';
```

Comment:
I tried multiple things to imitate what you did for the `core/styles.layer.css` but couldn't get it to work. It loses all styles when I change it to the layer 
I assume because its a different mantine package (tiptap), I tried adding it to the exports in package.json and adding it to imports in index.css but couldn't get it to work 🤦‍♀️